### DISCUSSION TITLE
Game crashes when trying to hide/show any category.

### DISCUSSION THREAD

**Discussion by Indying** [shedaniel/RoughlyEnoughItems#1331] [Q & A]
Anytime I try clicking something like this...
![image](https://user-images.githubusercontent.com/68744595/213032896-78978fc4-0d39-41f2-92ea-60ecf08d5d03.png)
My game just says "Saving World" and closes.

Im using 1.19.3 Fabric with MultiMC and these are my mods.
![image](https://user-images.githubusercontent.com/68744595/213033309-caf89c81-5f8d-4e46-98db-fe7207f5fd1f.png)

**Comment by shedaniel** [comment]
Hey, thanks for creating a discussion to RoughlyEnoughItems. To help trace the issue down, please include the necessary logs

You should upload the `latest.log` from the `.minecraft/logs` folder that is next to your `mods` folder.
